src/pages/patientView/PatientViewPage.tsx

```diff
@@ -418,7 +418,7 @@ export default class PatientViewPage extends React.Component<IPatientViewPagePro
                                     <LoadingIndicator isLoading={patientViewPageStore.mutationData.isPending || patientViewPageStore.uncalledMutationData.isPending || patientViewPageStore.oncoKbAnnotatedGenes.isPending || patientViewPageStore.studyIdToStudy.isPending} />
 
                                     {
-                                        (patientViewPageStore.oncoKbAnnotatedGenes.isComplete && patientViewPageStore.mutationData.isComplete && patientViewPageStore.uncalledMutationData.isComplete && !!sampleManager && patientViewPageStore.studyIdToStudy.isComplete) && (
+                                        (patientViewPageStore.oncoKbAnnotatedGenes.isComplete && patientViewPageStore.mutationData.isComplete && patientViewPageStore.uncalledMutationData.isComplete && patientViewPageStore.clinicalDataGroupedBySampleMap.isComplete && !!sampleManager && patientViewPageStore.studyIdToStudy.isComplete && patientViewPageStore.clinicalDataGroupedBySampleMap.isComplete) && (
```

Comment:
small request but would you mind breaking this up into multiple lines? I know you were just following the existing format so it's fine if you leave it, I just think it might be easier to read.